indexer/services/ender/__tests__/handlers/order-fills/order-handler.test.ts

```diff
@@ -1472,30 +1472,52 @@ describe('OrderHandler', () => {
       'via knex',
       false,
       IndexerOrder_TimeInForce.TIME_IN_FORCE_UNSPECIFIED,
+      true,
     ],
     [
       'limit',
       'via SQL function',
       true,
       IndexerOrder_TimeInForce.TIME_IN_FORCE_UNSPECIFIED,
+      true,
+    ],
+    [
+      'post-only best effort canceled',
+      'via knex',
+      false,
+      IndexerOrder_TimeInForce.TIME_IN_FORCE_POST_ONLY,
+      true,
     ],
     [
-      'post-only',
+      'post-only best effort canceled',
+      'via SQL function',
+      true,
+      IndexerOrder_TimeInForce.TIME_IN_FORCE_POST_ONLY,
+      true,
+    ],
+    [
+      'post-only canceled',
       'via knex',
       false,
       IndexerOrder_TimeInForce.TIME_IN_FORCE_POST_ONLY,
+      false,
+      OrderStatus.CANCELED,
     ],
     [
-      'post-only',
+      'post-only canceled',
       'via SQL function',
       true,
       IndexerOrder_TimeInForce.TIME_IN_FORCE_POST_ONLY,
+      false,
+      OrderStatus.CANCELED,
     ],
   ])('correctly sets status for short term %s orders (%s)', async (
     _orderType: string,
     _name: string,
     useSqlFunction: boolean,
     timeInForce: IndexerOrder_TimeInForce,
+    isBestEffortCanceled: boolean, // either best effort canceled or canceled
+    status: OrderStatus = OrderStatus.BEST_EFFORT_CANCELED,
```

Comment:
nit: Better to add a single param (`status`) and then set in either cache as appropriate.